hack/generate/samples/internal/ansible/testdata/tasks/finalizerconcurrencytest_test.yml

```diff
@@ -0,0 +1,58 @@
+---
+# TODO(asmacdo) this should be the only task. the other is getting magiced in
+- name: Create the test.example.com/v1alpha1.FinalizerConcurrencyTest
```

Comment:
is there a reason the file is `finalizerconcurrencytest_test`? vs `finalizerconcurrency_test`?

Reply:
Yes, but its not a good one. The testdata generation takes the Kind name "FinalizerConcurrenctTest" and automagically creates a verify task finalizerconcurrencytest_test. Rather than delete that and put mine in place, I just clobbered it.